### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jinbpark**
- confidential_ml.md

### CODE REVIEW

File: docs/component/attestation.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+To address this problem, ARM CCA has been designed from the beginning, having RA in mind, and comes with a formal document ([ARM CCA Security Model](https://developer.arm.com/documentation/DEN0096/latest)). On top of it, the attestation token format and the architecture described in that document align well with *[Remote ATtestation procedureS (RATS)](https://datatracker.ietf.org/wg/rats/about/)* specification, which is in active development to standardize RA stuff. This is a good thing as it implies that RA of CCA is not tightly coupled with a specific protocol, rather can connect to any attestation protocol.
+
+## Report
+
+An attestation report (shortly, *report*) is an *evidence* produced by *attester* and consumbed by *verifier*. In ARM CCA, report consists of two different tokens:
+
+- *CCA Platform token*: it is used to assure that *attester* is running on a secure CCA platform. It covers the measurements of CCA platform components (e.g., RMM and EL3M) and wheter it is in debug state.
```
Comment (zpzigi754) [inline docs/component/attestation.md:12]: [a minor typo]
wheter -> whether

File: docs/component/attestation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Appraisal policy
+
+According to RATS, there is a term named *Appraisal Policy (shortly, Policy)*, which is central to how to build a real-world service on top of RA. You can basically think of Policy as a set of rules that *verifier* wants to enforce.
+For example, what tokens in a report say is basically sort of measurements signed by secure cryptographic keys. So, to build a meaningful security service around it, you have to write down Policy like "(1) Realm must have the measurement of X, (2) the measurement of CCA Platform software must be Y".
+
+As you may notice, managing Policy is out of scope of CCA as this is inherently not depedent on CCA. Instead, there are several open-source projects that take on this capability, for example, [Veraison](https://github.com/veraison/) and [Certifier Framework](https://github.com/vmware-research/certifier-framework-for-confidential-computing/). They all aim to impelement a standardized way to express and enforce Policy. (note that they sometimes are treated as a unified verification service as they are able to work across various TEEs)
```
Comment (zpzigi754) [inline docs/component/attestation.md:22]: [a minor type]
impelement -> implement

File: docs/component/certifier.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Certifier framework
+
+[Certifier framework](https://github.com/vmware-research/certifier-framework-for-confidential-computing) is a framework designed to bring the ability to handle Policy into reality in an TEE-agnostic way. It offers a set of client APIs that can plug into various TEEs including ISLET (ARM CCA) and server called Ceritifer Service, which takes the role of verifying attestation reports that come from various TEEs.
+ISLET currently adopts this framework to realize end-to-end confidential services that likely involve more than two different TEEs.
```
Comment (zpzigi754) [inline docs/component/certifier.md:4]: [a minor fix]
likely involve more than two different TEEs -> are likely to involve two or more different TEEs

File: docs/component/attestation.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Attestation
+
+Remote Attestation (RA) is the key of confidential computing platform, which is basically a method that convinces to *verifier* that a program (*attester*) is running on a proper confidential computing platform (e.g., SGX or ARM CCA).
+Unfortunately, at the beginning of ARM TrustZone which has been widely adopted by mobile device vendors up to this date, it lacks the support of RA in the form of formal specification. Some research papers (e.g., [SecTEE](https://dl.acm.org/doi/10.1145/3319535.3363205)) have proposed a method to bring RA into TrustZone. However, due to the lack of standardization, RA comes in vendor-specific forms.
```
Comment (zpzigi754) [inline docs/component/attestation.md:4]: [a minor fix]
formal specification -> a specification (or a requirement)

File: docs/component/attestation.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+To address this problem, ARM CCA has been designed from the beginning, having RA in mind, and comes with a formal document ([ARM CCA Security Model](https://developer.arm.com/documentation/DEN0096/latest)). On top of it, the attestation token format and the architecture described in that document align well with *[Remote ATtestation procedureS (RATS)](https://datatracker.ietf.org/wg/rats/about/)* specification, which is in active development to standardize RA stuff. This is a good thing as it implies that RA of CCA is not tightly coupled with a specific protocol, rather can connect to any attestation protocol.
```
Comment (zpzigi754) [inline docs/component/attestation.md:6]: [a minor fix]
formal document -> white paper (or just a document)

**Review by zpzigi754 — COMMENTED** [review]
Good to update a document. I've left minor fixes.

**Comment by jinbpark** [comment]
@zpzigi754 sorry for confusion..  this was a draft PR which should have been deleted.. (not match with the current form of the github page) I will delete this PR, but reflect your comment with a new PR. (in addition to other minor stuff that have to be done prior to CC summit)

**Comment by bitboom** [comment]
@zpzigi754 
Welcome to Islet :)
If you have other comments or want to add things to document, feel fee to update our `doc/`. It will be very helpful to our :smile:.
You can test the doc site on local host like https://github.com/Samsung/islet/pull/91.

**Pull request closed without merging**